web/packages/teleport/src/SessionRecordings/view/SshPlayer.tsx

```diff
@@ -56,6 +68,17 @@ export default function Player({
   const isPlaying = playerStatus === StatusEnum.PLAYING;
   const isComplete = isError || playerStatus === StatusEnum.COMPLETE;
 
+  const moveToTime = useCallback(
+    (newTime: number) => {
+      tty.move(newTime);
+    },
+    [tty]
+  );
+
+  useImperativeHandle(ref, () => ({
+    moveToTime,
```

Comment:
I'm assuming that the reason for using an imperative handle is the performance optimization that you mentioned in the PR description, right?

Reply:
Can you put a comment here so the information is available to future readers instead of hiding in a PR description? :-)

Reply:
Yep added